app/src/main/java/org/kiwix/kiwixmobile/search/AutoCompleteAdapter.java

```diff
@@ -34,17 +38,16 @@
 import org.kiwix.kiwixmobile.data.ZimContentProvider;
 import org.kiwix.kiwixmobile.utils.SharedPreferenceUtil;
 
-public class AutoCompleteAdapter extends ArrayAdapter<String> implements Filterable {
-
+public class AutoCompleteAdapter extends RecyclerView.Adapter<AutoCompleteAdapter.ViewHolder> implements Filterable {
   @Inject JNIKiwix currentJNIReader;
   @Inject
   SharedPreferenceUtil sharedPreferenceUtil;
-  private List<String> mData;
+  List<String> mData;
```

Comment:
Why have you changed its scope to default?

Reply:
I think I did that because of my habit of working on small, school projects. I'll change it.